vectordb_bench/backend/cases.py

```diff
@@ -88,6 +88,20 @@ def filters(self) -> dict | None:
 
         return None
 
+    @property
+    def overall_timeout(self) -> int:
+        if self.dataset.data.size <= 1_000_000:
+            return 60 * 60 * 8 # 8h
+        else:
+            return 60 * 60 * 24 # 24h
+
+    @property
+    def optimize_timeout(self) -> int:
+        if self.dataset.data.size <= 1_000_000:
+            return 60 * 60 * 1 # 1h
+        else:
+            return 60 * 60 * 8 # 8h
+
```

Comment:
Can we binding timeout with specific case? Since different dataset, dim can also affect the performance